command-parser.js

```diff
@@ -7,7 +7,7 @@
  *
  * Individual commands are put in:
  *   commands.js - "core" commands that shouldn't be modified
- *   config/commands.js - other commands that can be safely modified
+ *   chat-plugins/info.js - other commands that can be safely modified
  *
  * The command API is (mostly) documented in config/commands.js
```

Comment:
`adding-custom-commands.MD` ?
Might be a bit long. I'm not sure how much it matters because more than likely the first place they're going to look is `commands.js` which would point them to the right file. So if the name is a little obtuse it shouldn't be too much of a problem.